example/android/build.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+    repositories {
+        jcenter()
+        maven {
+            url "https://maven.google.com"
+        }
+    }
+
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.3'
+    }
+}
+
+allprojects {
+    repositories {
+        jcenter()
+        maven {
+            url "https://maven.google.com"
+        }
+    }
+}
+
+rootProject.buildDir = '../build'
+subprojects {
+    project.buildDir = "${rootProject.buildDir}/${project.name}"
+    project.evaluationDependsOn(':app')
+}
+
+task clean(type: Delete) {
+    delete rootProject.buildDir
+}
```

Comment:
please don't check in the .jar files or the gradlew and gradlew.bat files

Reply:
Sorry I put the comment nearest the files in the diff. It wasn't a comment on build.gradle. Github won't let me comment on the actual binary files.

The patch includes the gradle wrapper jar file and the shell scripts. It should not.

Reply:
Thanks, I removed the gradle dir and gradlew files